### PR TITLE
[Keyboard] add keyboard rart75, rart67 and rartpad

### PR DESCRIPTION
**Pull request by alabahuy**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by alabahuy** [comment]
adding new firmware for my keyboard design

File: keyboards/rart/rart67/info.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+    "keyboard_name": "RART67",
+    "url": "",
+    "maintainer": "Alabahuy",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                /* Row 1 */
```
Comment (fauxpark) [inline keyboards/rart/rart67/info.json:10]: This JSON file is not valid.

File: keyboards/rart/rart67/config.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID		0xFEED  // "AL"
+#define PRODUCT_ID 		0x5667  // "67"
```
Comment (fauxpark) [inline keyboards/rart/rart67/config.h:24]:
```suggestion
#define VENDOR_ID		0x414C  // "AL"
#define PRODUCT_ID 		0x0067  // "67"
```

File: keyboards/rart/rart67/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# RART 67
+
+65% pcb  
+
+Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+Hardware Supported: RART67 PCB, Mini USB
+Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rart67/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RART67 PCB, Mini USB
* Hardware Availability: Private GB
```

File: keyboards/rart/rart67/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make RART/rart67:default
```
Comment (fauxpark) [inline keyboards/rart/rart67/readme.md:11]:
```suggestion
    make rart/rart67:default
```

File: keyboards/rart/rart67/rules.mk [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
```
Comment (fauxpark) [inline keyboards/rart/rart67/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

File: keyboards/rart/rart67/rules.mk [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/rart/rart67/rules.mk:33]:
```suggestion
```

File: keyboards/rart/rart67/rules.mk [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+LAYOUTS = all
```
Comment (fauxpark) [inline keyboards/rart/rart67/rules.mk:35]:
```suggestion
```

File: keyboards/rart/rart75/config.h [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID		0xFEED  // "AL"
+#define PRODUCT_ID 		0x6675  // "75"
```
Comment (fauxpark) [inline keyboards/rart/rart75/config.h:24]:
```suggestion
#define VENDOR_ID		0x414C  // "AL"
#define PRODUCT_ID 		0x0075  // "75"
```

File: keyboards/rart/rart75/info.json [hunk 9]
```diff
@@ -0,0 +1,542 @@
+{
+    "keyboard_name": "RART75",
+    "url": "",
+    "maintainer": "Alabahuy",
+    "width": 16,
+    "height": 6,
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                /* Row 1 */
```
Comment (fauxpark) [inline keyboards/rart/rart75/info.json:10]: As above.

File: keyboards/rart/rart75/readme.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# RART 75
+
+75% pcb with encoder
+
+Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+Hardware Supported: RART75 PCB, Mini USB, Encoders
+Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rart75/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RART75 PCB, Mini USB, Encoders
* Hardware Availability: Private GB
```

File: keyboards/rart/rart75/readme.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make RART/rart75:default
```
Comment (fauxpark) [inline keyboards/rart/rart75/readme.md:11]:
```suggestion
    make rart/rart75:default
```

File: keyboards/rart/rart75/rules.mk [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
```
Comment (fauxpark) [inline keyboards/rart/rart75/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

File: keyboards/rart/rart75/rules.mk [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/rart/rart75/rules.mk:33]:
```suggestion
```

File: keyboards/rart/rart75/rules.mk [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+ENCODER_ENABLE = yes
+
+
+LAYOUTS = all ansi
```
Comment (fauxpark) [inline keyboards/rart/rart75/rules.mk:37]:
```suggestion
```

File: keyboards/rart/rartpad/config.h [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID		0xFEED  // "AL" 
+#define PRODUCT_ID 		0x4520	// "PAD"
```
Comment (fauxpark) [inline keyboards/rart/rartpad/config.h:24]:
```suggestion
#define VENDOR_ID		0x414C  // "AL" 
#define PRODUCT_ID 		0x0050	// "P"
```

File: keyboards/rart/rartpad/info.json [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+  "keyboard_name": "RARTPAD",
+  "url": "",
+  "maintainer": "Alabahuy",
+  "width": 5,
+  "height": 4,
+  "layouts": {
+    "LAYOUT_pad": {
+      "key_count": 17,
+	       "layout": [
+                /* Row 1 */
```
Comment (fauxpark) [inline keyboards/rart/rartpad/info.json:11]: As above.

File: keyboards/rart/rartpad/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_all(
+		KC_NLCK, KC_PSLS, KC_PAST, KC_TRNS, \
+		KC_P7,   KC_P8,   KC_P9,   KC_PEQL, \
+		KC_P4,   KC_P5,   KC_P6,   KC_PPLS, \
+		KC_P1,   KC_P2,   KC_P3,   KC_PMNS, \
+		MO(1),   KC_0,    KC_PDOT, KC_PENT\
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/default/keymap.c:27]:
```suggestion
		KC_NLCK, KC_PSLS, KC_PAST, KC_TRNS,
		KC_P7,   KC_P8,   KC_P9,   KC_PEQL,
		KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
		KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
		MO(1),   KC_0,    KC_PDOT, KC_PENT
```

File: keyboards/rart/rartpad/keymaps/default/keymap.c [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+	),
+  
+	[1] = LAYOUT_all(
+		KC_TRNS, RGB_HUI, RGB_HUD,  KC_TRNS, \
+		RGB_SAI, RGB_SAD, KC_MNXT,  KC_MPRV, \
+		RGB_VAI, RGB_VAD, KC_MSTP,  KC_MPLY, \
+		KC_COPY, KC_PSTE, KC_MYCM,  KC_CALC, \
+		KC_TRNS, RGB_TOG, RGB_RMOD, RGB_MOD  \
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/default/keymap.c:35]:
```suggestion
		KC_TRNS, RGB_HUI, RGB_HUD,  KC_TRNS,
		RGB_SAI, RGB_SAD, KC_MNXT,  KC_MPRV,
		RGB_VAI, RGB_VAD, KC_MSTP,  KC_MPLY,
		KC_COPY, KC_PSTE, KC_MYCM,  KC_CALC,
		KC_TRNS, RGB_TOG, RGB_RMOD, RGB_MOD
```

File: keyboards/rart/rartpad/keymaps/numpad/keymap.c [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_pad(
+		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, \
+		KC_P7,   KC_P8,   KC_P9,   KC_PPLS, \
+		KC_P4,   KC_P5,   KC_P6,            \
+		KC_P1,   KC_P2,   KC_P3,   KC_PENT, \
+		KC_P0,            KC_PDOT           \
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/numpad/keymap.c:27]:
```suggestion
		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
		KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
		KC_P4,   KC_P5,   KC_P6,
		KC_P1,   KC_P2,   KC_P3,   KC_PENT,
		KC_P0,            KC_PDOT
```

File: keyboards/rart/rartpad/readme.md [hunk 20]
```diff
@@ -0,0 +1,13 @@
+# RARTPAD
+
+5x4 Macropad pcb with encoder
+
+Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+Hardware Supported: RARTPAD PCB, Promicro, Mini USB, Encoders
+Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rartpad/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RARTPAD PCB, Promicro, Mini USB, Encoders
* Hardware Availability: Private GB
```

File: keyboards/rart/rartpad/readme.md [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make RART/rartpad:default
```
Comment (fauxpark) [inline keyboards/rart/rartpad/readme.md:11]:
```suggestion
    make rart/rartpad:default
```

File: keyboards/rart/rartpad/rules.mk [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+
+
+LAYOUTS = all pad
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rules.mk:30]:
```suggestion
```

File: keyboards/rart/rart67/info.json (hunk 1 above)
Comment (alabahuy) [reply keyboards/rart/rart67/info.json]: i already to fix it

File: keyboards/rart/rart67/config.h (hunk 2 above)
Comment (alabahuy) [reply keyboards/rart/rart67/config.h]: i already to fix it

File: keyboards/rart/rart67/readme.md (hunk 3 above)
Comment (alabahuy) [reply keyboards/rart/rart67/readme.md]: i already to fix it

File: keyboards/rart/rart67/readme.md [hunk 23]
```diff
@@ -0,0 +1,13 @@
+# RART 67
+
+65% pcb  
+
+*Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+*Hardware Supported: RART67 PCB, Mini USB
+*Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rart67/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RART67 PCB, Mini USB
* Hardware Availability: Private GB
```

File: keyboards/rart/rart75/readme.md [hunk 24]
```diff
@@ -0,0 +1,13 @@
+# RART 75
+
+75% pcb with encoder
+
+*Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+*Hardware Supported: RART75 PCB, Mini USB, Encoders
+*Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rart75/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RART75 PCB, Mini USB, Encoders
* Hardware Availability: Private GB
```

File: keyboards/rart/rartpad/readme.md [hunk 25]
```diff
@@ -0,0 +1,13 @@
+# RARTPAD
+
+5x4 Macropad pcb with double encoder
+
+*Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)  
+*Hardware Supported: RARTPAD PCB, Promicro, Mini USB, Encoders
+*Hardware Availability: Private GB
```
Comment (fauxpark) [inline keyboards/rart/rartpad/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
* Hardware Supported: RARTPAD PCB, Promicro, Mini USB, Encoders
* Hardware Availability: Private GB
```

File: keyboards/rart/rartpad/rules.mk [hunk 26]
```diff
@@ -0,0 +1,31 @@
...
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+
+
+LAYOUTS = pad all
+
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rules.mk:31]:
```suggestion
```

File: keyboards/rart/rart67/readme.md (hunk 23 above)
Comment (alabahuy) [reply keyboards/rart/rart67/readme.md]: done

File: keyboards/rart/rartpad/rules.mk (hunk 26 above)
Comment (alabahuy) [reply keyboards/rart/rartpad/rules.mk]: i already to fix it

File: keyboards/rart/rart75/readme.md (hunk 24 above)
Comment (alabahuy) [reply keyboards/rart/rart75/readme.md]: i already to fix it

File: keyboards/rart/rartpad/rules.mk [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no  	# Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rules.mk:28]:
```suggestion

LAYOUTS = ortho_5x4 numpad_5x4
```

File: keyboards/rart/rartpad/rartpad.h [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_all( \
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rartpad.h:14]:
```suggestion
#define LAYOUT_ortho_5x4( \
```

File: keyboards/rart/rartpad/rartpad.h [hunk 29]
```diff
@@ -0,0 +1,41 @@
...
+	K40, K41, K42, K43  \
+) \
+{ \
+	{ K00,   K01,   K02,   K03 }, \
+	{ K10,   K11,   K12,   K13 }, \
+	{ K20,   K21,   K22,   K23 }, \
+	{ K30,   K31,   K32,   K33 }, \
+	{ K40,   K41,   K42,   K43 }, \
+}
+#define LAYOUT_pad( \
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rartpad.h:28]:
```suggestion

#define LAYOUT_numpad_5x4( \
```

File: keyboards/rart/rartpad/keymaps/numpad/keymap.c [hunk 30]
```diff
@@ -0,0 +1,45 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_pad(
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/numpad/keymap.c:22]:
```suggestion
	[0] = LAYOUT_numpad_5x4(
```

File: keyboards/rart/rartpad/keymaps/default/keymap.c [hunk 31]
```diff
@@ -0,0 +1,54 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/default/keymap.c:22]:
```suggestion
	[0] = LAYOUT_ortho_5x4(
```

File: keyboards/rart/rartpad/keymaps/default/keymap.c [hunk 32]
```diff
@@ -0,0 +1,54 @@
...
+		KC_NLCK, KC_PSLS, KC_PAST, KC_TRNS,
+		KC_P7,   KC_P8,   KC_P9,   KC_PEQL,
+		KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
+		KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
+		MO(1),   KC_0,    KC_PDOT, KC_PENT
+	),
+  
+	[1] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/default/keymap.c:30]:
```suggestion
	[1] = LAYOUT_ortho_5x4(
```

File: keyboards/rart/rartpad/rartpad.h [hunk 33]
```diff
@@ -0,0 +1,41 @@
...
+	K00, K01, K02, K03, \
+	K10, K11, K12, K13, \
+	K20, K21, K22,      \
+	K30, K31, K32, K33, \
+	K40,      K42      \
```
Comment (fauxpark) [inline keyboards/rart/rartpad/rartpad.h:33]:
```suggestion
	K10, K11, K12,      \
	K20, K21, K22, K13, \
	K30, K31, K32,      \
	K40,      K42, K33, \
```

File: keyboards/rart/rartpad/keymaps/numpad/keymap.c [hunk 34]
```diff
@@ -0,0 +1,45 @@
...
+		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, 
+		KC_P7,   KC_P8,   KC_P9,   KC_PPLS, 
+		KC_P4,   KC_P5,   KC_P6,            
+		KC_P1,   KC_P2,   KC_P3,   KC_PENT, 
+		KC_P0,            KC_PDOT
```
Comment (fauxpark) [inline keyboards/rart/rartpad/keymaps/numpad/keymap.c:27]:
```suggestion
		KC_P7,   KC_P8,   KC_P9,
		KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
		KC_P1,   KC_P2,   KC_P3,
		KC_P0,            KC_PDOT, KC_PENT,
```

File: keyboards/rart/rartpad/info.json [hunk 35]
```diff
@@ -0,0 +1,140 @@
...
+                {
+                  "label" : ".",
+                  "x":2, "y":4},
+                {
+                  "label" : "Enter",
+                  "x":3, "y":4},         
+                ]
+   }
+  }
+}
```
Comment (fauxpark) [inline keyboards/rart/rartpad/info.json:140]:
```suggestion
{
    "keyboard_name": "RARTPAD",
    "url": "",
    "maintainer": "Alabahuy",
    "width": 5,
    "height": 4,
    "layouts": {
        "LAYOUT_numpad_5x4": {
            "layout": [
                {"label": "Numlock", "x": 0, "y": 0},
                {"label": "/", "x": 1, "y": 0},
                {"label": "*", "x": 2, "y": 0},
                {"label": "-", "x": 3, "y": 0},

                {"label": "7", "x": 0, "y": 1},
                {"label": "8", "x": 1, "y": 1},
                {"label": "9", "x": 2, "y": 1},

                {"label": "4", "x": 0, "y": 2},
                {"label": "5", "x": 1, "y": 2},
                {"label": "6", "x": 2, "y": 2},
                {"label": "+", "x": 3, "y": 1, "h": 2},

                {"label": "1", "x": 0, "y": 3},
                {"label": "2", "x": 1, "y": 3},
                {"label": "3", "x": 2, "y": 3},

                {"label": "0", "x": 0, "y": 4, "w": 2},
                {"label": ".", "x": 2, "y": 4},
                {"label": "Enter", "x": 3, "y": 3, "h": 2}
            ]
        },
        "LAYOUT_ortho_5x4": {
            "layout": [
                {"label": "Numlock", "x": 0, "y": 0},
                {"label": "/", "x": 1, "y": 0},
                {"label": "*", "x": 2, "y": 0},
                {"label": "-", "x": 3, "y": 0},

                {"label": "7", "x": 0, "y": 1},
                {"label": "8", "x": 1, "y": 1},
                {"label": "9", "x": 2, "y": 1},
                {"label": "=", "x": 3, "y": 1},

                {"label": "4", "x": 0, "y": 2},
                {"label": "5", "x": 1, "y": 2},
                {"label": "6", "x": 2, "y": 2},
                {"label": "+", "x": 3, "y": 2},

                {"label": "1", "x": 0, "y": 3},
                {"label": "2", "x": 1, "y": 3},
                {"label": "3", "x": 2, "y": 3},
                {"label": "Esc", "x": 3, "y": 3},

                {"label": "0", "x": 0, "y": 4},
                {"label": "00", "x": 1, "y": 4},
                {"label": ".", "x": 2, "y": 4},
                {"label": "Enter", "x": 3, "y": 4}
            ]
        }
    }
}
```

Comment (alabahuy) [reply keyboards/rart/rartpad/info.json]: done to  fix  this

File: keyboards/rart/rartpad/rartpad.h (hunk 33 above)
Comment (alabahuy) [reply keyboards/rart/rartpad/rartpad.h]: i try to fix this but always fail.  where is the error? I have followed the instructions

Comment (fauxpark) [reply keyboards/rart/rartpad/rartpad.h]: Whoops sorry. Fixed.

Comment (alabahuy) [reply keyboards/rart/rartpad/rartpad.h]: done

Comment (alabahuy) [reply keyboards/rart/rartpad/rartpad.h]: dont  know but  always error in LAYOUT_numpad_5x4

Comment (alabahuy) [reply keyboards/rart/rartpad/rartpad.h]: solved, thanks

File: keyboards/rart/rartpad/rartpad.h (hunk 28 above)
Comment (alabahuy) [reply keyboards/rart/rartpad/rartpad.h]: done

File: keyboards/rart/rartpad/keymaps/numpad/keymap.c (hunk 34 above)
Comment (alabahuy) [reply keyboards/rart/rartpad/keymaps/numpad/keymap.c]: done

File: keyboards/rart/rartpad/keymaps/default/keymap.c (hunk 32 above)
Comment (alabahuy) [reply keyboards/rart/rartpad/keymaps/default/keymap.c]: Is there no follow up for my request?

**Comment by noroadsleft** [comment]
@alabahuy Did you have a problem of some sort here? Why was this closed?

**Comment by alabahuy** [comment]
@noroadsleft I think I opened too many requests, I tried to make one after another,
Am I wrong? I apologize

**Comment by noroadsleft** [comment]
Ah, I see you've opened #9543.

We do generally allow multiple keyboards to be added at once if all the keyboards are by the same person, but in my opinion it's better when you're starting out to just stick to one thing per pull request.

**Pull request closed without merging**